workspace/apps/pidp/src/app/features/shell/components/navbar-menu/nav-menu.ts

```diff
@@ -79,21 +86,41 @@ export class NavMenuComponent implements OnChanges {
   public isLogoutMenuItemVisible = false;
   public isTopMenuVisible = false;
   public ProfileRoutes = ProfileRoutes;
+  public AccessRoutes = AccessRoutes;
   public showCollegeAlert = false;
   public faBell = faBell;
   public AlertCode = AlertCode;
+  public credentials: Credential[] = [];
+  public credentials$: Observable<Credential[]>;
+  private unsubscribe$ = new Subject<void>();
+  public IdentityProvider = IdentityProvider;
 
   public constructor(
     private viewportService: ViewportService,
     private router: Router,
+    private resource: NavMenuResource,
+    private partyService: PartyService,
   ) {
     this.viewportService.viewportBroadcast$.subscribe((viewport) =>
       this.onViewportChange(viewport),
     );
+    const partyId = this.partyService.partyId;
+    this.credentials$ = this.resource.getCredentials(partyId);
   }
+
+  public ngOnInit(): void {
+      this.handleLinkedAccounts();
+  }
+
   public ngOnChanges(_: SimpleChanges): void {
     this.refresh();
   }
+
+  public ngOnDestroy(): void {
```

Comment:
We need the class `NavMenuComponent` to implement `OnDestroy` to call this life cycle method.